src/scaffoldmaker/meshtypes/meshtype_3d_stellate1.py

```diff
@@ -367,8 +465,36 @@ def generateBaseMesh(cls, region, options):
                         element = mesh.createElement(elementIdentifier, elementtemplate1)
                         result = element.setNodesByIdentifier(eft1, nodeIdentifiers)
                         result3 = element.setScaleFactors(eft1, scalefactors) if scalefactors else None
+
+                        # add to meshGroup
+                        stellateMeshGroup.addElement(element)
+                        if isMouse:
+                            if na == 0:
+                                arm1MeshGroup.addElement(element)
+                            elif na == 1:
+                                arm2MeshGroup.addElement(element)
+                            elif na == 2:
+                                arm3MeshGroup.addElement(element)
+
+                        ############################
+                        # annotation fiducial points
+                        ############################
+                        if allMarkers:
+                            for key in allMarkers:
+                                if elementIdentifier == allMarkers[key]["elementID"]:
+                                    elementID = allMarkers[key]["elementID"]
+                                    xi = allMarkers[key]["xi"]
+                                    addMarker = {"name": key, "xi": allMarkers[key]["xi"]}
+
+                                    markerPoint = markerPoints.createNode(nodeIdentifier, markerTemplateInternal)
+                                    nodeIdentifier += 1
+                                    cache.setNode(markerPoint)
+                                    markerName.assignString(cache, addMarker["name"])
+                                    markerLocation.assignMeshLocation(cache, element, addMarker["xi"])
```

Comment:
Need to discuss if there's a better way to do this, especially if we locate markers by proportions over each arm. (I typically try to avoid having loops overall markers for all elements.)
I'm thinking the elements loop builds arrays of n*2 element identifies for each arm, then have a subsequent step outside the loop for making marker points on it.